packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -614,3 +615,60 @@ export interface GetIndexerMakerStatisticsResponse {
   rewardCoefficient: BigDecimal;
   makers: IndexerMaker[];
 }
+
+export interface GetIndexerLeaderboardParams {
+  contestId: number;
+  rankType: IndexerLeaderboardRankType;
+  // Min rank inclusive
+  startCursor?: string;
+  limit?: number;
+}
+
+export interface IndexerLeaderboardParticipant {
+  subaccount: Subaccount;
+  contestId: number;
+  pnl: BigDecimal;
+  pnlRank: BigDecimal;
+  percentRoi: BigDecimal;
+  roiRank: BigDecimal;
+  // Seconds
+  updateTime: BigDecimal;
+}
+
+export interface GetIndexerLeaderboardResponse {
+  participants: IndexerLeaderboardParticipant[];
+}
+
+export interface GetIndexerLeaderboardParticipantParams {
+  contestId: number;
+  subaccount: Subaccount;
+}
+
+export interface GetIndexerLeaderboardParticipantResponse {
+  participant: IndexerLeaderboardParticipant | null;
```

Comment:
nit: can we leave a comment saying that this is null if not eligible?

Reply:
addressed